repology/parsers/parsers/sisyphus.py

```diff
@@ -0,0 +1,38 @@
+from typing import Iterable
```

Comment:
Lacks GPL header. Please make sure not to copypaste my name there.

Reply:
done